.github/workflows/ci.yaml

```diff
@@ -70,14 +70,12 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      - name: Vendor dependencies
-        run: go mod vendor
-      - uses: bufbuild/buf-setup-action@v1
+      - name: Install linter
+        uses: bufbuild/buf-setup-action@v1
         with:
           version: "1.20.0"
-      - uses: bufbuild/buf-lint-action@v1
-        with:
-          input: api
+      - name: Run linter
+        run: make lint-proto
```

Comment:
The reason I used `buf-lint-action` was to annotate message to the lines with error 🤔 (Same as the golangci-lint)

I agree with your concerns - How about adding `make lint-proto` step at the end for make sure the job fails when the one of `Makefile` known input fails?

Reply:
You've reminded me... Go linting step needs this same improvement.

Not sure I entirely understood your comment. Can I propose that you are welcome to amend this PR with your suggestion?

Reply:
Sure!